### PR TITLE
Handle ValidationException and throw HttpStatusException

### PR DESCRIPTION
**Pull request by niketh**
@DennisMcWherter @wcekan Can you review

### CODE REVIEW

File: elide-core/src/main/java/com/yahoo/elide/Elide.java [hunk 1]
```diff
@@ -244,6 +245,14 @@ protected ElideResponse handleRequest(boolean isReadOnly, Object opaqueUser,
             return buildErrorResponse(new InvalidURLException(e), isVerbose);
 
         } catch (Exception e) {
+            if (e instanceof ValidationException) {
+                return buildErrorResponse(
+                    new HttpStatusException(HttpStatus.SC_BAD_REQUEST, e.getMessage()) {
```
Comment (DennisMcWherter) [inline elide-core/src/main/java/com/yahoo/elide/Elide.java]: I would check first if there exists `statusCode` on the `ValidationException` and return that as an HTTP response?

**Comment by DennisMcWherter** [comment]
Any update on this?

File: elide-core/src/main/java/com/yahoo/elide/Elide.java [hunk 2]
```diff
@@ -244,6 +245,14 @@ protected ElideResponse handleRequest(boolean isReadOnly, Object opaqueUser,
             return buildErrorResponse(new InvalidURLException(e), isVerbose);
 
         } catch (Exception e) {
+            if (e instanceof ValidationException) {
```
Comment (wcekan) [inline elide-core/src/main/java/com/yahoo/elide/Elide.java]: Why is this not
```java
catch (ValidationException e)
```

**Comment by DennisMcWherter** [comment]
I'm going to close for now due to inactivity. Feel free to reopen!

**Pull request closed without merging**